### PR TITLE
Docker mods in progress

### PR DESCRIPTION
**Pull request by areedv**
Move some tasks out and improve build process speed of this project

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
-    libssl-dev
-
-# basic R functionality
-RUN R -e "install.packages(c('remotes'), repos='https://cloud.r-project.org/')"
-
-# install qmongr app
-RUN R -e "remotes::install_github('SKDE-Felles/qmongr', upgrade = 'never')"
+
+# install package dependency from github
+RUN R -e "remotes::install_github('SKDE-Felles/qmongrdata', upgrade = 'always')"
```
Comment (arnfinn) [inline Dockerfile]: Can 
```
remotes::install_github('SKDE-Felles/qmongrdata', upgrade = 'always')
```
be replaced by something like
```
devtools::install_deps(".")
```
?

Comment (areedv) [reply Dockerfile]: In fact, the sole purpose is to make sure the latest version of _qmongrdata_ will be present when building the _qmongr_ docker image. Everything else should be taken care of by the [initial docker image](https://github.com/SKDE-Felles/qmongr-base-r/blob/master/Dockerfile). To ensure a shortest possible build time _upgrade_ should be set to 'never' here as well

File: .travis.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@ r_packages:
   - devtools
 
 before_install:
+  - R CMD build .
```
Comment (arnfinn) [inline .travis.yml]: I guess this is on purpose, to install it both here and in the `script` session?

Comment (areedv) [reply .travis.yml]: Yes, practically no time cost...

**Pull request merged**